testing/cuda/reduce_by_key.cu

```diff
@@ -286,3 +289,103 @@ void TestReduceByKeyCudaStreamsNoSync()
 }
 DECLARE_UNITTEST(TestReduceByKeyCudaStreamsNoSync);
 
+
+// Maps indices to key ids
+class div_op : public thrust::unary_function<long long, long long>
```

Comment:
Consider using `std::int64_t` instead of `long long` to emphasize that this must be a 64-bit type.